public/js/components/SourceTabs.js

```diff
@@ -171,21 +171,6 @@ const SourceTabs = React.createClass({
     }, filename);
   },
 
-  renderSourcesDropdownButton() {
```

Comment:
This looks like it wasn't being called anywhere else and is just being removed here.